script/deploy/deploy-testnet-assets.s.sol

```diff
@@ -146,78 +148,95 @@ abstract contract AssetDeployer is Script {
         return IUniswapV3Factory(uniFactory).createPool(token0, token1, feeTier);
     }
 
-    function initializePool(address poolAddress, AssetPair memory pair, address deployer) internal {
-        CollarOwnedERC20 collateralAsset = CollarOwnedERC20(deployedAssets[pair.collateralSymbol]);
+    function getSqrtPriceX96ForPriceRatio(uint amount0, uint amount1) internal pure returns (uint160) {
+        // Just use the raw ratio for sqrtPrice calculation
+        // Price is amount1/amount0 in Q96 format
+        return uint160(Math.sqrt((amount1 << 192) / amount0));
```

Comment:
this needs an explanation of what this is doing and why, and possibly links to helpful resources for further reading